css/site.css

```diff
@@ -108,3 +108,47 @@
     }
   }
 }
+
+@media (prefers-color-scheme: dark) {
+  body {
+    background-color: rgb(0, 0, 0);
+    color: rgb(255, 255, 255);
+  }
+
+  .card-title.text-truncate,
+  .navbar-brand {
+    color: rgb(255, 255, 255) !important;
```

Comment:
Does it work without `!important`? How about increasing the specificity if it doesn't work without `!important`?